### PR TITLE
Add HTTP_PROXIES related parameters

### PR DESCRIPTION
**Pull request by threes33**
Related Issue: N/A

## New Behavior

Configure `HTTP_PROXIES` settings for http/https scheme through environment variable using `HTTP_PROXY` and `HTTPS_PROXY`

## Contrast to Current Behavior

There is currently no support to configure this through an environment variable so a custom configuration must be created

## Discussion: Benefits and Drawbacks

Makes it easier to set http proxies

## Changes to the Wiki

N/A

## Proposed Release Note Entry

Support the following environment variables for `HTTPS_PROXIES`:

- `HTTP_PROXY`
- `HTTPS_PROXY`

Also see: https://netboxlabs.com/docs/netbox/en/stable/configuration/system/#http_proxies

## Double Check

* [x] I have read the comments and followed the PR template.
* [x] I have explained my PR according to the information in the comments.
* [x] My PR targets the `develop` branch.

### CODE REVIEW

File: configuration/configuration.py [hunk 1]
```diff
@@ -200,10 +200,10 @@ def _environ_get_and_map(variable_name: str, default: str | None = None, map_fn:
...
 
 # HTTP proxies NetBox should use when sending outbound HTTP requests (e.g. for webhooks).
-# HTTP_PROXIES = {
-#     'http': 'http://10.10.1.10:3128',
-#     'https': 'http://10.10.1.10:1080',
-# }
+HTTP_PROXIES = {
+    'http': environ.get('HTTP_PROXY', None),
+    'https': environ.get('HTTPS_PROXY', None),
+}
```
Comment (cimnine) [inline configuration/configuration.py:206]: Have you tested that this works for those that don't need any proxy?

**Review by cimnine — COMMENTED** [review]
Thanks for the PR. Please see my question below.

File: configuration/configuration.py (hunk 1 above)
Comment (threes33) [reply configuration/configuration.py:206]: Hi @cimnine - I did acknowledge this was working by confirming the "Netbox News" widget loaded WITHOUT any proxy env set and netbox system having direct access to internet.

This panel gives a timeout if proxy is needed for internet access so is retrieved by netbox itself

Comment (cimnine) [reply configuration/configuration.py:206]: Thanks for the confirmation!

**Review by tobiasge — APPROVED** [review]

**Pull request merged**